src/primitives/primitive_style.rs

```diff
@@ -42,6 +42,13 @@ where
     /// This property only applies to closed shapes (rectangle, circle, ...) and is
     /// ignored for open shapes (line, ...).
     pub stroke_alignment: StrokeAlignment,
+
+    /// Stroke style.
+    ///
+    /// The `stroke_style` sets a particular style, for instance the dotted style.
+    /// This feature is only implemented for the [`Rectangle`](crate::primitives::Rectangle)
+    /// primitive, and `stroke_style` will be ignored for all other primitives.
+    pub stroke_style: Option<StrokeStyle>,
```

Comment:
There is no need to use an `Option` here. Just add a `Solid` stroke style and make that the default.